de/add-external-libraries.md

```diff
@@ -38,19 +38,19 @@ Wähle das zu deinem Computer passende System und folge den angegebenen Schritte
 
 1. Die meisten externen Libraries findest du in Github-Repositories. Um sie herunterzuladen, musst du den grünen Button `Clone or download` und daraufhin im sich öffnenden Fenster `Download ZIP` klicken. 
 
-  ![Beispielhafter Download der "Adafruit_HDC_Library" von github.com](pictures/libraries/github_download.png)
+  ![Beispielhafter Download der "Adafruit_HDC_Library" von github.com]https://github.com/sensebox/resources/raw/master/gitbook_pictures/libraries/github_download.png)
```

Comment:
Is there a bracket missing here before the image URL? Also in the other links of this file.